Move, shrink and grow windows key bindings

cyberang3l opened the pull request:
Added 6 new key bindings to grow/shrink and move windows around (up/down/left/right).

This PR depends on #41.

I usually use `Alt+NumPad6` or `Alt+NumPad7` to place windows right or left, but if I have
more than two windows the entire desktop is covered and I cannot see all windows.

With the functionality added in this PR, I use the existing key bindings (e.g., `Alt+NumPad7`
or `Ctrl+NumPad9`) for the main placement of the window, and then I can shrink/grow the
windows in place (anchored to the center of the window that is being shrunk/grown) and
move them around to micro-place them and size them exactly where I want.

You can see a screencast of the functionality here:
[Screencast_20240309_160931.webm](https://github.com/lucmos/UltrawideWindows/assets/5658474/7e9c95db-4989-44f8-9b1e-ec8d0dc5e844)

All the window movements in the screencast are happening by the newly added
key binding, i.e., no mouse involved.

cyberang3l committed:
Added functionality to shrink/grow and move windows left/right/up/down

Registers the following additional shortcuts:

* IncreaseWindowSize: Ctrl+Meta+Num++
* DecreaseWindowSize: Ctrl+Meta+Num+-
* MoveWindowLeft:     Ctrl+Meta+Left
* MoveWindowRight:    Ctrl+Meta+Right
* MoveWindowUp:       Ctrl+Meta+Up
* MoveWindowDown:     Ctrl+Meta+Down

cyberang3l committed:
Updated README

lucmos commented:
Looks great!

Could you rebase --onto master to remove the commit already merged with #41?

cyberang3l committed:
Metadata.desktop is not needed any more - remove

The metadata.json file is the recommended way to go as per the latest
KWIN scripting tutorial:

https://develop.kde.org/docs/plasma/kwin/

cyberang3l committed:
Remove some dead code
